github/code-scanning.go

```diff
@@ -115,6 +115,12 @@ type AlertListOptions struct {
 
 	// Return code scanning alerts for a specific branch reference. The ref must be formatted as heads/<branch name>.
 	Ref string `url:"ref,omitempty"`
+
+	// Returns the results based on Page number. Page number is set to 1 by default
+	Page int `url:"page,omitempty"`
+
+	// Results per page. Maximum is 100 results page
+	PerPage int `url:"per_page,omitempty"`
```

Comment:
```suggestion
	ListOptions
```

Reply:
@gmlewis I'm not understanding properly why we are removing the 2 entries and replacing it with `ListOptions`. Can you please explain?

Reply:
> @gmlewis I'm not understanding properly why we are removing the 2 entries and replacing it with `ListOptions`. Can you please explain?

In Go, this is called "type embedding" or "struct embedding".  Since `ListOptions` provides all we need here, we aren't duplicating the code every place we need these fields.  Take a look through our repo, and this is how we embed these list options in other options.

Reply:
Thanks for the detailed explanation! @gmlewis